test/90_1_prm/test_appendix_g_prm.rb

```diff
@@ -1363,12 +1368,27 @@ def check_if_sz_vav(model, zone)
     return zone_system_check
   end
 
+  def check_multi_bldg_handling(baseline_base)
+    baseline_base.each do |baseline, model_baseline|
+      building_type, template, climate_zone, user_data_dir, mod = baseline
+      # Get WWR of baseline model
+      wwr_baseline = run_query_tabulardatawithstrings(model_baseline, 'InputVerificationandResultsSummary', 'Conditioned Window-Wall Ratio', 'Gross Window-Wall Ratio', 'Total', '%').to_f
+
+      # Check WWR against expected WWR
+      wwr_goal = 100 * @@wwr_values[building_type].to_f
+      assert(wwr_baseline < wwr_goal, "Baseline WWR for the #{building_type}, #{template}, #{climate_zone} model with user data is incorrect. The WWR of the baseline model is #{wwr_baseline} but should be #{wwr_baseline}, smaller than the WWR goal #{wwr_goal}")
+
+      # TODO adding more tests to check if zones are assigned correctly
+    end
+
+  end
```

Comment:
I think it would be good to add a second test where we find two HVAC building area types. Maybe use large hotel and differentiate between guest rooms and public areas.